charts/spire/README.md

```diff
@@ -68,6 +68,14 @@ helm upgrade --install -n spire-mgmt spire spire --repo https://spiffe.github.io
 
 We only support upgrading one major version at a time. Version skipping isn't supported.
 
+## Clean up
```

Comment:
Didn't notice before, but the cleanup section is nested here inside the middle of the upgrade notes section. It needs to be moved out to its own section.